jboss-kie-wildfly-common/tests/bats/expectations/formatter/standalone-expected.xml

```diff
@@ -472,7 +472,7 @@
       <stacks>
         <stack name="udp">
           <transport type="UDP" socket-binding="jgroups-udp"/>
-          <protocol type="openshift.DNS_PING"/>
+          <!-- ##JGROUPS_PING_PROTOCOL## -->
```

Comment:
why add this?

Reply:
It's a placeholder used by the script to add the configuration https://github.com/desmax74/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch/jgroups/added/launch/ha.sh#L32

Reply:
ok, thanks.